app/controllers/users_controller.rb

```diff
@@ -1,5 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
```

Comment:
- [ ] since you are using the user#index please make sure that the view is rendering the posts.

Reply:
I have this code in my show method:
```
  def show
    @user = User.find(params[:id])
  end
```
and I can see it is rendering the data for a specific User.